deploy-as-code/helm/charts/ifix-services/pspcl-ifix-adapter/values.yaml

```diff
@@ -0,0 +1,62 @@
+# Common Labels
+labels:
+  app: "pspcl-ifix-adapter"
+  group: "ifix"
+
+cron:
+  schedule: "0 0 * * *"  
+
+# Init Containers Configs
+initContainers:
+  dbMigration:
+    enabled: false
+
+# Container Configs
+image:
+  repository: "pspcl-ifix-adapter"
+
+cron_appType: "java-spring"
+dbMigration:
+  schemaTable: "pspcl_ifix_adapter_schema"
+pspcl-ifix-datasource: "pspcl_ifix"
```

Comment:
Which data source does this refer to? 
I can't see any reference to this in the environment variables below. 

At this place, we usually define default values of the variables that have to be defined in env files. 
If we want to make some variables mandatory, we should not provide any default value so that the deployment fails if someone has failed to set that env var in the env file.

Reply:
this is not required..i'll remove